apis/falcon/v1alpha1/falconcontainer_types.go

```diff
@@ -1,53 +1,114 @@
-	Injector *FalconContainerInjectorSpec `json:"injector,omitempty"`
+	// +kubebuilder:default:={imagePullPolicy:Always}
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Injector Configuration",order=3
+	Injector FalconContainerInjectorSpec `json:"injector,omitempty"`
 
 	// Falcon Container Version. The latest version will be selected when version specifier is missing.
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Image Version",order=4
 	Version *string `json:"version,omitempty"`
 }
 
-// Represents the status of Falcon deployment
-type FalconContainerStatusPhase string
+type FalconContainerInjectorSpec struct {
+	// Define annotations that will be passed down to injector service account. This is useful for passing along AWS IAM Role or GCP Workload Identity.
+	// +kubebuilder:default:={name:default}
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Service Account Configuration",order=1
+	ServiceAccount FalconContainerServiceAccount `json:"serviceAccount,omitempty"`
+
+	// +kubebuilder:default:=4433
+	// +kubebuilder:validation:XIntOrString
+	// +kubebuilder:validation:Minimum:=0
+	// +kubebuilder:validation:Maximum:=65535
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Injector Listen Port",order=2
+	ListenPort *int32 `json:"listenPort,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Injector TLS Configuration",order=3
+	TLS FalconContainerInjectorTLS `json:"tls,omitempty"`
+
+	// +kubebuilder:default:=Always
+	// +kubebuilder:validation:Enum=Always;IfNotPresent;Never
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Image Pull Policy",order=4
+	ImagePullPolicy corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
+
+	// +kubebuilder:default=crowdstrike-falcon-pull-secret
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Image Pull Secret Name",order=5
+	ImagePullSecretName string `json:"imagePullSecret,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Shared Log Volume",order=6
+	LogVolume *corev1.Volume `json:"logVolume,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Injector Resources",order=7
+	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Sensor Resources",order=8
+	SensorResources *corev1.ResourceRequirements `json:"sensorResources,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falconctl Opts options",order=9
+	FalconctlOpts string `json:"falconctlOpts,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Additional Environment Variables",order=10
```

Comment:
This is really just intended to support 0-day releases of the container sensor, where new inputs might be defined.  We'll be tracking the inputs for the container sensor, so new options are added, we'll seek to add them explicitly to our CRDs within a short period of time -- but this will allow for customers to leverage those new options before we issue a new release and they are able to update their operator deployment.